lighthouse-core/audits/deprecations.js

```diff
@@ -30,10 +31,572 @@ const UIStrings = {
+  /**
+  *@description TODO(crbug.com/1318876): Description needed for translation
+  */
+  rtpDataChannel:
+  'RTP data channels are no longer supported. The `RtpDataChannels` constraint is currently ignored, and may cause an error at a later date.',
+  /**
+  *@description TODO(crbug.com/1320361): Description needed for translation
+  */
+  selectionAddRangeIntersect:
+  'The behavior that `Selection.addRange()` merges existing Range and the specified Range was removed.',
+  /**
+  *@description TODO(crbug.com/1318878): Description needed for translation
+  */
+  sharedArrayBufferConstructedWithoutIsolation:
+  '`SharedArrayBuffer` will require cross-origin isolation. See https://developer.chrome.com/blog/enabling-shared-array-buffer/ for more details.',
+  /**
+  *@description TODO(crbug.com/1320363): Description needed for translation
+  */
+  textToSpeech_DisallowedByAutoplay:
+  '`speechSynthesis.speak()` without user activation is deprecated and will be removed.',
+  /**
+  *@description TODO(crbug.com/1318879): Description needed for translation
+  */
+  v8SharedArrayBufferConstructedInExtensionWithoutIsolation:
+  'Extensions should opt into cross-origin isolation to continue using `SharedArrayBuffer`. See https://developer.chrome.com/docs/extensions/mv3/cross-origin-isolation/.',
+  /**
+  *@description TODO(crbug.com/1318880): Description needed for translation
+  */
+  webCodecsVideoFrameDefaultTimestamp:
+  'Constructing a `VideoFrame` without a timestamp is deprecated and support will be removed. Please provide a timestamp via `VideoFrameInit`.',
+  /**
+  *@description TODO(crbug.com/1318881): Description needed for translation
+  */
+  xhrJSONEncodingDetection: 'UTF-16 is not supported by response json in `XMLHttpRequest`',
+  /**
+  *@description TODO(crbug.com/1318882): Description needed for translation
+  */
+  xmlHttpRequestSynchronousInNonWorkerOutsideBeforeUnload:
+  'Synchronous `XMLHttpRequest` on the main thread is deprecated because of its detrimental effects to the end user\u2019s experience. For more help, check https://xhr.spec.whatwg.org/.',
+  /**
+  *@description TODO(crbug.com/1320365): Description needed for translation
+  */
+  xrSupportsSession:
+  '`supportsSession()` is deprecated. Please use `isSessionSupported()` and check the resolved boolean value instead.',
+  /** ... */
+  unknownDeprecation: '...',
 };
+/* eslint-enable max-len */
 
 const str_ = i18n.createMessageInstanceIdFn(__filename, UIStrings);
 
+/**
+ * @param {LH.Crdp.Audits.DeprecationIssueDetails} deprecation
+ */
+function getMeta(deprecation) {
+  let message;
+  let feature = 0;
+  let milestone = 0;
+  // @ts-expect-error
+  const legacyMessage = /** @type {string=} */ (deprecation.message);
+
+  switch (deprecation.type) {
+    case 'AuthorizationCoveredByWildcard':
+      message = str_(UIStrings.authorizationCoveredByWildcard);
+      milestone = 97;
```

Comment:
If deprecations are added/changed we are going to need to update this like `InspectorIssues` right?